src/libmongoc/doc/mongoc_client_pool_set_bind_ip.rst

```diff
@@ -0,0 +1,32 @@
+:man_page: mongoc_client_pool_set_bind_ip
+
+mongoc_client_pool_set_bind_ip()
```

Comment:
If we only offered a client-level setter, then there would be a gap between when the first outgoing connections are made in the driver and when the user could set a custom bind_ip.  So, monitoring might not use the custom bind_ip right away.  Offering a pool-level setter avoids this gap.